{"uuid": "e12518a0-9390-496b-8eeb-c500f458cc62", "children": ["72b835ee-89a5-4fed-a0f4-2378d621b88d"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1750919781136, "stop": 1750919781136}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\JS492UT\\PycharmProjects\\PlaywrightLearning\\.venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\JS492UT\\PycharmProjects\\PlaywrightLearning\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\JS492UT\\PycharmProjects\\PlaywrightLearning\\testcases\\conftest.py\", line 37, in log_on_failure\n    if item.rep_call.failed:\n"}, "start": 1750919781150, "stop": 1750919781155}], "start": 1750919781136, "stop": 1750919781155}